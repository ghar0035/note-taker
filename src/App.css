
.lnkBtn {
  color: #333;
  display: inline-block;
  padding: 0.4rem 1rem;
  border: 2px solid #333;
  border-radius: 4px;
  text-decoration: none;
  text-transform: capitalize;
}

.lnkBtn.primary {
  background-color: #333;
  color: #fff;
}
