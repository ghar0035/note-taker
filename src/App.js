import styled from "styled-components";
import "./App.css";
import Header from "./components/common/header";
import noteBanner from "./images/note_banner.png";

const Banner = styled.div`
  padding: 2rem;
  display: flex;
  height: 80vh;
  align-items: center;
  justify-content: space-around;

  .img {
    max-width: 30rem;
    height: auto;
    border-radius: 1rem;
  }
`;

const Footer = styled.div`
padding: 1rem;
`;

function App() {
  return (
    <div className="App">
      <Header />
      <div className="container">
        <Banner>
          <div>
            <h1>
              Note - the app that <br /> simplifies your note-taking!
            </h1>

            <button className="btn">Try it now!</button>
          </div>

          <div>
            <img className="img" alt="" src={noteBanner} />
          </div>
        </Banner>
      </div>
      <Footer>
        <p>all rights reserved &copy;</p>
      </Footer>
    </div>
  );
}

export default App;
