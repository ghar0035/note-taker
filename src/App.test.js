import renderer from "react-test-renderer";
import App from "./App";
import { MemoryRouter as Router } from "react-router-dom";

test("renders App", () => {
  const tree = renderer
    .create(
      <Router>
        <App />
      </Router>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
