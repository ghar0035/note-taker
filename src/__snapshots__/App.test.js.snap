// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App 1`] = `
<div
  className="App"
>
  <div
    className="sc-bcXHqe evUqyv"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      <img
        alt="logo"
        className="logo"
        src="logo.png"
      />
    </a>
    <div>
      <a
        className="lnkBtn"
        href="/auth/register"
        onClick={[Function]}
      >
        register
      </a>
      <a
        className="lnkBtn primary"
        href="/auth"
        onClick={[Function]}
      >
        login
      </a>
    </div>
  </div>
  <div
    className="container"
  >
    <div
      className="sc-gswNZR hyxNuc"
    >
      <div>
        <h1>
          Note - the app that 
          <br />
           simplifies your note-taking!
        </h1>
        <button
          className="btn"
        >
          Try it now!
        </button>
      </div>
      <div>
        <img
          alt=""
          className="img"
          src="note_banner.png"
        />
      </div>
    </div>
  </div>
  <div
    className="sc-dkrFOg iBfJNu"
  >
    <p>
      all rights reserved Â©
    </p>
  </div>
</div>
`;
